pkg/cmd/util/clientcmd/clientconfig_test.go

```diff
@@ -0,0 +1,18 @@
+package clientcmd
+
+import (
+	"testing"
+
+	"github.com/spf13/pflag"
+)
+
+func TestDefaultShorthands(t *testing.T) {
+	flags := pflag.NewFlagSet("test", pflag.ExitOnError)
```

Comment:
you ought to be able to call their `DefaultClientConfig` with a `flagset` too and then make sure the same flags are set.  There may be a special case or two to whitelist.